elastic/kibana
# [Console] History view enhancements

## Summary

This PR moves the history modal into the `history` view. Besides that, there are also a few visual enhancements to how it looks and we've added an option to add and execute the restored command.

##### How to test

* Start up es and kibana
* Navigate to `Devtools`
* Verify that:
  * When executing a command, it's added to the history
  * History commands are sorted by date
  * When a command is restored, it scrolls down in the editor to it and highlights it in blue
  * When a command is restored and executed, it scrolls down in the editor to it, highlights it in blue and executes it.


https://github.com/user-attachments/assets/31c5e53c-a478-4e25-a510-bafa545420ad


<details>
<summary>UI screenshots</summary>

![Screenshot 2024-08-30 at 13 45 34](https://github.com/user-attachments/assets/bc8cd286-67a5-4410-9cb3-672e093bd79a)


</details>
